test/lint/lint-cppcheck-dash.py

```
@@ -0,0 +1,147 @@
+        '-DCLIENT_VERSION_IS_RELEASE',
+        '-DCLIENT_VERSION_MAJOR',
+        '-DCLIENT_VERSION_MINOR',
+        '-DCOPYRIGHT_YEAR',
+        '-DDEBUG',
+        '-DUSE_EPOLL',
+        '-DCHAR_BIT=8',
+        '-I', 'src/',
+        '-q',
+    ] + files
+
+    dependencies_output = subprocess.run(
```

Ruff (0.13.3)

`subprocess` call: check for execution of untrusted input

(S603)